include/dictionary-search.php

```diff
@@ -361,17 +360,38 @@ function replace_default_search_filter($input, $query=null)
 	if (empty($query))
 	    return $input;
 
-	if (is_search() && (isset($_GET['s']) || isset($query->query_vars['s'])))
+	// chungh 20200303 I re-write the logic slightly for clarity.
+	// But in doing so, I realized that COMBINED searches, e.g. searching for a word
+	// within semantic domain or parts of speech (taxomony) would not work as is.
+	// Nor does paging, as all results are shown on every page.
+	// TODO: fix these in Webonary 1.5		
+	if (isset($_GET['tax']) && $_GET['tax'] > 1)
+	{
+		$input = "SELECT DISTINCTROW $wpdb->posts.* " .
```

Comment:
I left $input in there, because as I noted in the comment above, this whole block of code is not working. $input should be built progressively, depending on the value of each parms.  Will clean up if we end up trying to fix this bug further